test/unit/stream_alert_rule_processor/test_classifier.py

```diff
@@ -13,148 +13,270 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 '''
-
-# command: nosetests -v -s test/unit/
-# specific test: nosetests -v -s test/unit/file.py:TestStreamPayload.test_name
-
-import base64
 import json
 
-from nose.tools import assert_equal, assert_not_equal
+from mock import call, patch
 
-import stream_alert.rule_processor.classifier as sa_classifier
+from nose.tools import (
+    assert_equal,
+    assert_false,
+    assert_is_instance,
+    assert_list_equal,
+    assert_true
+)
 
-from stream_alert.rule_processor.classifier import StreamPayload, StreamClassifier
-from stream_alert.rule_processor.pre_parsers import StreamPreParsers
+import stream_alert.rule_processor.classifier as sa_classifier
 from stream_alert.rule_processor.config import load_config
+from stream_alert.rule_processor.payload import load_stream_payload
 
+from unit.stream_alert_rule_processor.test_helpers import (
+    _make_kinesis_raw_record
+)
 
-class TestStreamPayload(object):
-    """Test class for StreamPayload"""
-    def __init__(self):
-        self.config = {}
 
+class TestStreamClassifier(object):
+    """Test class for StreamClassifier"""
     @classmethod
     def setup_class(cls):
         """Setup the class before any methods"""
-        cls.env = {
-            'lambda_region': 'us-east-1',
-            'account_id': '123456789012',
-            'lambda_function_name': 'test_kinesis_stream',
-            'lambda_alias': 'production'
-        }
+        config = load_config('test/unit/conf')
+        cls.classifier = sa_classifier.StreamClassifier(config)
```

Comment:
Because the classifier stores generated state (namely `self._entity_log_sources`), I would suggest creating the StreamClassifier for every test (i.e. in `setup`, not `setup_class`). This will help avoid weird bugs with persistent state between different tests